13_sandboxes/sandbox_pool.py

```diff
@@ -0,0 +1,231 @@
+# ---
+# cmd: ["python", "13_sandboxes/sandbox_pool.py"]
+# pytest: false
+# ---
+
+# # Build a pool of warm sandboxes that are healthy and ready to serve requests
```

Comment:
nit: we like to use proper noun form for our products. So `Sandboxes` not `sandboxes`.